skills/gh-address-comments/scripts/pr_changelog.py

```
@@ -0,0 +1,227 @@
+import argparse
+import json
+import re
+import subprocess
+import sys
+from collections import defaultdict
+from typing import Any
+
+# Matches: "Resolves review thread PRRT_kwDO..."
+_THREAD_REF_RE = re.compile(r"Resolves review thread (PRRT_\S+)", re.IGNORECASE)
+# Also match: "address PR comment #N - description"
+_COMMENT_REF_RE = re.compile(r"address PR comment #(\d+)\s*[-–]\s*(.+)", re.IGNORECASE)
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)